packages/gitbeaker-core/src/index.ts

```diff
@@ -1,6 +1,15 @@
 import { bundler } from './infrastructure';
 import * as APIServices from './services';
 
+/* -------------- Export Map ------------- */
+export function getAPIMap(): Record<string, unknown> {
+  if (!'__apiMap__'.includes('{')) {
+    throw new Error('This function is only available in the distributed code');
+  }
+
+  return JSON.parse('__apiMap__');
```

Comment:
shouldn't `__apiMap__` be a global variable instead of a string? That's how it works with webpack, but not sure about rollup.

Reply:
The rollup version does a string replace at compile time